### PR TITLE
Edit view

### PR DESCRIPTION
**Pull request by justalak**
https://edu-redmine.sun-asterisk.vn/issues/21913

### CODE REVIEW

**Comment by justalak** [comment]
ready

File: app/views/comments/_comment.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="comments-block" id="comment-<%= comment.id %>">
+<div class="comments-block" %>">
```
Comment (manhbnt-0960) [inline app/views/comments/_comment.html.erb]: thừa %>

File: app/views/comments/_comment_detail.html.erb [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+<div class="comment-wrapper" id="comment-<%= comment.id %>">
+  <div class="comment-text <%= "reply-comments" unless comment.root? %>">
+    <p class="comment-content" id="content-<%= comment.id %>">
+      <%= render "comments/comment_content", comment: comment %>
+    </p>
+    <% if current_user? comment.user %>
+      <div class="dropdown">
+        <button class="dropbtn"><i class="fa fa-cog"></i></button>
+        <div class="dropdown-content">
+          <%= button_to post_comment_path(comment.post_id, comment.id),
```
Comment (manhbnt-0960) [inline app/views/comments/_comment_detail.html.erb]: indent

File: app/views/comments/_comment_detail.html.erb [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+        class: "delete-comment",
+        method: :delete,
+        remote: true,
+        data: { confirm: t("sure_delete")} do %>
+            <i class="fa fa-trash"></i>
+          <% end %>
+          <button data_id="<%= comment.id %>" class="edit-comment"
+        data-toggle="modal" data-target="#edit-comment">
```
Comment (manhbnt-0960) [inline app/views/comments/_comment_detail.html.erb]: indent

File: app/views/comments/_edit_comment_modal.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div class="text-center">
+          <div class="form-group">
+            <textarea class="form-control" id="comment-content"></textarea>
+            </div>
+            <button id="edit-submit" class="btn btn-primary ">
```
Comment (manhbnt-0960) [inline app/views/comments/_edit_comment_modal.html.erb]:
```suggestion
            <button id="edit-submit" class="btn btn-primary">
```

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -8,3 +9,7 @@
     <%= render  "shared/card_footer", post: @post %>
   </div>
 </div>
+<%= render "posts/edit_modal", post: @post %>
+<%= render "shared/list_modal" %>
+<%= render  "comments/edit_comment_modal" %>
```
Comment (manhbnt-0960) [inline app/views/posts/show.html.erb]:
```suggestion
<%= render "comments/edit_comment_modal" %>
```

**Comment by justalak** [comment]
ready

**Pull request merged**